src/components/form-elements/draft-js-mention-selector/DraftJSMentionSelectorCore.js

```diff
@@ -197,6 +202,11 @@ class DraftJSMentionSelector extends React.Component<Props, State> {
                 }
             },
         );
+
+        if (activeMention && activeMention.mentionString?.length > 0 && isFocused) {
```

Comment:
Instead of using a `setTimeout` to hide the a11y alert after showing it, would utilizing a unique for the alert span work? That way we force React to render a new DOM element instead of reusing it which should help solve the screen reader issue

Reply:
sorry I don't get it

Reply:
how would be the unique alert span syntax
`<span id=unique('someText')>` ?

Reply:
I guess it will not work since we have to wait for some miliseconds before set the value in false
I mean if I set 

`setTimeout(callback, 0); // this fail` 
`setTimeout(callback, 100); // this works`

Reply:
Sorry, I mis-typed, utilizing a unique `key` on the element, something like this:
https://codesandbox.io/s/a11y-alert-react-key-u04u4